notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb

```diff
@@ -0,0 +1,722 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip install keras-cv "keras==2.15" numpy "tensorflow==2.15.1" openvino-nightly gradio
</pre><p>please add -q flag to install command, there is also some issues with installing openvino-nightly on windows platform if user already have previously installed openvino package. For avoiding that we usually uninstall previously used openvino packages</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Fixed